seek (and thus continue where you left off) was
broken by commit cadcd7a9842a0b2439d24999b53cd75f53328dcf
diff --git a/modules/demux/mkv/matroska_segment_seeker.cpp b/modules/demux/mkv/matroska_segment_seeker.cpp
index 8feecb902f..666fe007fc 100644
--- a/modules/demux/mkv/matroska_segment_seeker.cpp
+++ b/modules/demux/mkv/matroska_segment_seeker.cpp
@@ -310,7 +310,7 @@ SegmentSeeker::get_seekpoints( matroska_segment_c& ms, mtime_t target_pts,
         if ( start.fpos == std::numeric_limits<fptr_t>::max() )
             return tracks_seekpoint_t();
 
-        if ( end.fpos != std::numeric_limits<fptr_t>::max() || !ms.b_cues )
+        if ( end.fpos != std::numeric_limits<fptr_t>::max() || !ms.b_cues || true)
             // do not read the whole (infinite?) file to get seek indexes
             index_range( ms, Range( start.fpos, end.fpos ), needle_pts );
 
